libs/cmdio/io.go

```diff
@@ -192,11 +186,19 @@ func (c *cmdIO) Select(names map[string]string, label string) (id string, err er
 
 func Select[V any](ctx context.Context, names map[string]V, label string) (id string, err error) {
 	c := fromContext(ctx)
-	stringNames := map[string]string{}
+	var items []Tuple
 	for k, v := range names {
-		stringNames[k] = fmt.Sprint(v)
+		items = append(items, Tuple{k, fmt.Sprint(v)})
 	}
-	return c.Select(stringNames, label)
+	slices.SortFunc(items, func(a, b Tuple) int {
+		return strings.Compare(a.Name, b.Name)
+	})
+	return c.Select(items, label)
+}
+
+func SelectOrdered(ctx context.Context, items []Tuple, label string) (id string, err error) {
```

Comment:
Can you add one line doc strings to `Select` and `SelectOrdered`. It's not obvious from the name that `Select` sorts the items, while `SelectOrdered` prints the items in the sequence provided.

Reply:
Done